help/develop/getting-started.md

```diff
@@ -3,208 +3,215 @@
 
 # Getting Started with Building on Datagrok
 
-Welcome to [Datagrok](https://www.datagrok.ai), the next generation data platform!
+Welcome to Datagrok, the next-generation data platform!
 
-This is a guide to help you jump-start using Datagrok as an engineer, extending Datagrok
-with plugins and building applications on top of it. During this Getting Started,
-post your questions and suggestions at our [Community Forum](https://community.datagrok.ai/).
-We usually respond within less than a day.
+This guide will help you jump-start using Datagrok as an engineer. In short, what you can do with Datagrok is extend it
+with plugins or build your applications on top of Datagrok.
+
+If you have any questions or suggestions after reading this Getting Started guide, go to our [Community Forum] &mdash; 
+typically respond within 24 hours.
```

Comment:
Shouldn't there be a subject in this sentence, e.g., 'we respond'?

Reply:
Yep, it's a typo.